common/openshift_ops.py

```diff
@@ -0,0 +1,104 @@
+import logging
+from collections import defaultdict
+from kubernetes import config
+from openshift.dynamic import DynamicClient
+
+log = logging.getLogger(__name__)
+
+
+class OCP(object):
+    """Class which contains various utility functions for interacting
+    with OpenShift
+
+    """
+    def __init__(self):
+
+        k8s_client = config.new_client_from_config()
```

Comment:
Do we need a config file here?

Reply:
> Do we need a config file here?
No need of extra config file. It used ~/.kube/config